src/components/useraccount/MyOrders.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div v-if="me"
+       class="my-orders col-sm-9">
+    <div class="my-orders-content">
+      <div class="row">
+        <div class="col-sm-12">
+          <div class="my-orders-title hidden-xs">
+            <h4>{{$t ('myOrders')}}</h4>
+          </div>
+        </div>
+      </div>
+      <div class="my-orders-table-wrapper">
+        <div class="row">
+          <div class="col-sm-2 col-xs-4">
+            <div class="my-orders-table-titles">
+              <h5>{{$t ('orderNumber')}}</h5>
+            </div>
+          </div>
+          <div class="col-sm-2 hidden-xs">
+            <div class="my-orders-table-titles">
+              <h5>{{$t ('date')}}</h5>
+            </div>
+          </div>
+          <div class="col-sm-2 col-xs-4">
+            <div class="my-orders-table-titles">
+              <h5>{{$t ('total')}}</h5>
+            </div>
+          </div>
+          <div class="col-sm-2 hidden-xs">
+            <div class="my-orders-table-titles">
+              <h5>{{$t ('paymentStatus')}}</h5>
+            </div>
+          </div>
+          <div class="col-sm-2 hidden-xs">
+            <div class="my-orders-table-titles">
+              <h5>{{$t ('shipmentStatus')}}</h5>
+            </div>
+          </div>
+          <div class="col-sm-2 col-xs-4">
+            <div class="my-orders-table-titles">
+              &nbsp;
+            </div>
+          </div>
+        </div>
+      </div>
+      <div v-for="order in me.orders.results"
+           :key="order.id"
+           class="my-orders-table-order">
+        <div class="row">
+          <div class="col-sm-2 col-xs-4">
+            <span>{{ order.orderNumber || "-" }}</span>
+          </div>
+          <div data-test="order-date"
+               class="col-sm-2 hidden-xs">
+            <p>{{ $d(new Date(order.createdAt), 'short', 'en-US') }}</p>
```

Comment:
Seeing now this line, I strongly think we should create a `BaseDate` similar to what we had for prices. As props we would have the `date` as commercetools sends it (e.g. `order.createdAt`) and `format` (e.g. short/long/etc.).
Then inside the component we should get the country that the customer has selected, so that they can see the date in the format that is normal for their localization.
WDYT?